packages/nc-gui/components/dashboard/TreeView.vue

```diff
@@ -216,6 +216,22 @@ const onSearchCloseIconClick = () => {
   filterQuery = ''
   toggleSearchActive(false)
 }
+
+useEventListener(document, 'keydown', async (e: KeyboardEvent) => {
+  if (e.altKey) {
```

Comment:
what about `alt + ctrl`, `alt + shift`, `cmd + shift`, etc, check if those values are `false` or not. That way it will trigger for exact `ALT + T`,  at the moment it will even trigger for `cmd + alt + t` .